scripts/check-deps.sh

```diff
@@ -55,9 +55,9 @@ done
 # github.com/foo/bar github.com/foo/bar/v3) there will be a duplicate
 # in the list.  Remove duplicates again.
 mv "${tmpdir}/HEAD" "${tmpdir}/HEAD-dup"
-uniq "${tmpdir}/HEAD-dup" > "${tmpdir}/HEAD"
+uniq "${tmpdir}/HEAD-dup" | sort -n > "${tmpdir}/HEAD"
 
-grep '^-' docs/LICENSE_OF_DEPENDENCIES.md | grep -v github.com/DataDog/datadog-agent | cut -f 2 -d' ' > "${tmpdir}/LICENSE_OF_DEPENDENCIES.md"
+grep '^-' docs/LICENSE_OF_DEPENDENCIES.md | grep -v github.com/DataDog/datadog-agent | cut -f 2 -d' ' | sort -n > "${tmpdir}/LICENSE_OF_DEPENDENCIES.md"
```

Comment:
Why are these changes required for this PR? Seems odd to sort numerically.

Reply:
Ok, Thanks for your feedback, i have rebase and resolve the conflicts.

 It's diffcult to add `github.com/imdario/mergo` and `github.com/spf13/pflag` in the right place in the `LICENSE_OF_DEPENDENCIES.md`  because we need to ensure it in alphabetical order.

Reply:
Hmm `go mod tidy` did not handle that for you? I am still seeing a conflict.

Reply:
> @yahaa I'd prefer if we kept the dependency licenses in alphabetical order opposed to numerical, would you mind removing this change? We could always consider this in a separate pull request.

Sorry, forgot to reply you. I also think a single pr would be better.